mail/mail/doctype/mail_account/mail_account.py

```diff
@@ -164,6 +165,15 @@ def validate_display_name(self) -> None:
 		if self.is_new() and not self.display_name:
 			self.display_name = frappe.db.get_value("User", self.user, "full_name")
 
+	def validate_backup_email(self) -> None:
+		"""Validates the backup email."""
+
+		validate_email_address(self.backup_email, True)
+
+		if self.user != get_dmarc_address():
+			if self.backup_email in get_user_email_addresses(self.user):
```

Comment:
Not sure if this works for the new account as the account doc is not inserted means get_user_email_addresses will return []. Just try in local once.

Steps:
- Create a User.
- Create a Mail Account with backup_email = User.email

Reply:
https://github.com/frappe/mail/commit/ab761b164869782301166397ea8266940a532e91